{
  expect(mock.oneArg(false)).andReturn("Test").andReturn("Test2").andThrow(new IndexOutOfBoundsException()).atLeastOnce();
  replay(mock);
  assertEquals("Test",mock.oneArg(false));
  assertEquals("Test2",mock.oneArg(false));
  try {
    mock.oneArg(false);
  }
 catch (  IndexOutOfBoundsException expected) {
  }
  try {
    mock.oneArg(false);
  }
 catch (  IndexOutOfBoundsException expected) {
  }
}
